gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilSubplugin.kt

```diff
@@ -56,13 +57,18 @@ class AnvilSubplugin : KotlinGradleSubplugin<AbstractCompile> {
       val key = "${getCompilerPluginId()}.${srcGenDirOption.key}"
       val relativePath = srcGenDir.relativeTo(project.buildDir).path
 
-      project.tasks.withType(KotlinCompile::class.java) { task ->
-        task.inputs.property(key, relativePath)
+      project.tasks.withType(KotlinCompile::class.java).configureEach { task ->
+          task.inputs.property(key, relativePath)
       }
-      project.tasks.withType(KaptTask::class.java) { task ->
-        // We don't apply the compiler plugin for KaptTasks. But for some reason the Kotlin Gradle
-        // plugin copies the inputs with the "kotlinCompile" prefix.
-        task.inputs.property("kotlinCompile.$key", relativePath)
+      project.tasks.withType(KaptTask::class.java).configureEach { task ->
+          // We don't apply the compiler plugin for KaptTasks. But for some reason the Kotlin Gradle
+          // plugin copies the inputs with the "kotlinCompile" prefix.
+          task.inputs.property("kotlinCompile.$key", relativePath)
+      }
+      project.tasks.withType(KaptGenerateStubsTask::class.java).configureEach { task ->
```

Comment:
That's not needed. `KaptGenerateStubsTask` is a `KotlinCompile` task and handled above. With Kotlin 1.4 there isn't the extra key and there's no need to add the input property
```
task.inputs.property("kotlinCompile.$key", relativePath)
```
I tested exactly this yesterday.

Reply:
It fixed the error of the previous build...